src/seedu/addressbook/parser/Parser.java

```diff
@@ -111,7 +111,7 @@ public Command parseCommand(String userInput) {
      * @param args full command args string
      * @return the prepared command
      */
-    private Command prepareAdd(String args) {
+    private static Command prepareAdd(String args) {
```

Comment:
Instead of making all the other methods static, could you have created an instance of parser inside the `parseCommand` method to call these other methods instead?